doc/overview.rst

```diff
@@ -41,6 +41,18 @@ Presentations
 These are conference presentations about the development of PyGMT (previously
 "GMT/Python"):
 
+* "Geospatial Analysis & Visualization with PyGMT".
+  2022.
+  Max Jones, Wei Ji Leong, and Leonardo Uieda.
+  Presented at *SciPy 2022*.
+  doi:`10.6084/m9.figshare.20483793 <https://doi.org/10.6084/m9.figshare.20483793>`__
+
+.. figure:: _static/scipy2022-youtube-thumbnail.png
```

Comment:
> Nice presentation!
> 
> For this PR, I feel the thumbnail image `doc/_static/scipy2022-youtube-thumbnail.png` is too large (834 KB) and should not be included in the repo.

Could we source the thumbnail from Youtube directly (i.e. use a relative link)? E.g. following https://stackoverflow.com/questions/14192709/is-it-possible-to-embed-youtube-vimeo-videos-in-markdown-using-a-c-sharp-markdow. Downside is that places blocking Youtube might not be able to see the thumbnail :slightly_smiling_face:

Reply:
The thumbnail image (https://img.youtube.com/vi/nCktihu9bWg/0.jpg) is not as good as the screenshot.
![](https://img.youtube.com/vi/nCktihu9bWg/0.jpg)

Reply:
I followed the recommendation to compress using https://squoosh.app/. Thanks for the suggestion.